### PR TITLE
RDM-5309-refactored

### PR DESCRIPTION
**Pull request by akdasari**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: ccd/values.yaml [hunk 1]
```diff
@@ -149,37 +151,37 @@ draft-store-service:
...
-#       ASB_CONNECTION_STRING: false
-#       SPRING_DATASOURCE_USERNAME: hmcts
-#       SPRING_DATASOURCE_PASSWORD: hmcts
-#       SPRING_LIQUIBASE_ENABLED: true
-#       AUTH_IDAM_CLIENT_BASEURL: '{{ .Values.global.idamApiUrl }}'
-#       CORE_CASE_DATA_API_URL: 'http://ccd-data-store'
-#       SPRING_DATASOURCE_URL:  jdbc:postgresql://{{ .Release.Name }}-postgresql:5432/payment_api 
-#       AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: '{{ .Values.global.s2sUrl }}'
+payment-api:
+  enabled: true
```
Comment (timja) [inline ccd/values.yaml]: probably shouldn't be on by default?
also I think the readme needs updating?
https://github.com/hmcts/chart-ccd#chart-ccd

File: ccd/values.yaml [hunk 2]
```diff
@@ -149,37 +151,37 @@ draft-store-service:
...
+      SPRING_DATASOURCE_PASSWORD: '{{ .Values.global.postgresPassword }}'
+      SPRING_LIQUIBASE_ENABLED: true
+      AUTH_IDAM_CLIENT_BASEURL: '{{ .Values.global.idamApiUrl }}'
+      CORE_CASE_DATA_API_URL: 'http://ccd-data-store'
+      SPRING_DATASOURCE_URL:  jdbc:postgresql://{{ .Release.Name }}-postgresql:5432/payment_api 
+      AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: '{{ .Values.global.s2sUrl }}'
       
 
 dm-store:
+  enabled: true
```
Comment (timja) [inline ccd/values.yaml]: off by default maybe? to keep the default version only having 'core' ccd
and possibly have another file that enables all the flags that you need for your testing?

File: ccd/values.yaml (hunk 1 above)
Comment (adusumillipraveen) [reply ccd/values.yaml]: Lets keep Readme to end, when we get this working

File: ccd/values.yaml (hunk 2 above)
Comment (akdasari) [reply ccd/values.yaml]: left only core components as default now, moved enable options to ci-values and values-sandbox.yaml

File: ccd/values.yaml (hunk 1 above)
Comment (akdasari) [reply ccd/values.yaml]: Will update Readme once this is finalised

File: ccd/values.yaml [hunk 3]
```diff
@@ -67,15 +68,15 @@ ccd-api-gateway-web:
       PROXY_CASE_ACTIVITY: http://ccd-case-activity-api
     
 ccd-user-profile-api:
-  enabled: true
```
Comment (timja) [inline ccd/values.yaml]: user-profile should probably be on by default?

**Pull request merged**